projects/core/src/configurator/commons/facade/configurator-groups.service.ts

```diff
@@ -135,6 +97,33 @@ export class ConfiguratorGroupsService {
     );
   }
 
+  /**
+   * Determines the group status by the group ID and the switcher that defines whether the group has been visited or not.
```

Comment:
JS doc is not really understandable...because the method does not determine anything but just states that the group is visited

Reply:
This does not only check for group "visited", but also for "success" and "error" and sets the corresponding status in the state. This method is only a wrapper method for the corresponding group status service method. The only difference is, that I read the configuration, therefor I copied the JS doc.